waspc/data/Generator/templates/react-app/src/test/vitest/helpers.tsx

```diff
@@ -8,9 +8,7 @@ import { QueryClient, QueryClientProvider } from '@tanstack/react-query'
 import { beforeAll, afterEach, afterAll } from 'vitest'
 import { Query } from '../../queries'
 import config from '../../config'
-import { HttpMethod } from '../../types'
-
-export type Route = { method: HttpMethod; path: string }
```

Comment:
Should we re-export this from here for safety? Unsure if I used it in docs or anywhere else.

Reply:
Reexported 👍